libmozdata/utils.py

```diff
@@ -381,3 +382,39 @@ def get_language(path):
     for lang, names in langs.items():
         if name in names or extension in names:
             return lang
+
+
+# This is a roughly equivalent to the itertools.islice which was added in
+# Python 3.10.
```

Comment:
What if we just require 3.10?

Reply:
I prefer that. We need first to update Taskcluster to run on Python 3.10 instead of Python 3.9.

I filed an issue for that since it is out of the scope of this PR: https://github.com/mozilla/libmozdata/issues/245